sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -942,21 +947,24 @@ def get_manifest(self, repository: str, tag_or_digest: str, **kwargs) -> GetMani
                 **kwargs
             )
         )
+        manifest_size = _get_manifest_size(response.http_response.headers)
```

Comment:
So the content-length header will always represent the _total_ manifest size? Not just the chunk?

Reply:
Yes. - manifest is not downloading by chunks, it is downloading in one time. So the content-length header represent the content size in this download call, which is the total manifest size.

Reply:
Ah yes - I was confusing it with the blob download :P